winui/Rating/Customization.md

```diff
@@ -9,4 +9,242 @@ documentation: ug
 
-This section explains the customization features available in the WinUI Rating control.
+This section explains the customization features available in the WinUI `Rating` control.
+
+## Style
+
+The Rating control style can be customized using the `RatedItemStyle`, `UnratedItemStyle` Property.
+
+{% tabs %}
+{% highlight XAML %}
+
+<ResourceDictionary>
+ <Style TargetType="Path" x:Key="ratedStyle">
+    <Setter Property="Fill" Value="#FFD700"/>
+    <Setter Property="Stroke" Value="#FFD700"/>
+    <Setter Property="StrokeThickness" Value="1"/>
+ </Style>
+ <Style TargetType="Path" x:Key="unratedStyle">   
+    <Setter Property="Fill" Value="LightGray"/>
+    <Setter Property="Stroke" Value="LightGray"/>
+    <Setter Property="StrokeThickness" Value="1"/>
+ </Style>
+</ResourceDictionary>
+
+<syncfusion:SfRating Value="3"
+      ItemsCount="5" RatedItemStyle={StaticResource ratedStyle}
+      UnratedItemStyle={StaticResource unratedStyle}>
+</syncfusion:SfRating>
+
+{% endhighlight %}
+{% highlight C# %}
+
+//RatedItemStyle
+Style style1 = new Style(typeof(Path));
+Setter setter1 = new Setter();
+setter1.Property = Path.FillProperty;
+setter1.Value = new SolidColorBrush(Color.FromArgb(255, 255, 215, 0));
+style1.Setters.Add(setter1);
+
+Setter setter2 = new Setter();
+setter2.Property = Path.StrokeProperty;
+setter2.Value =new SolidColorBrush(Color.FromArgb(255, 255, 215, 0));
+style1.Setters.Add(setter2);
+
+Setter setter3 = new Setter();
+setter3.Property = Path.StrokeThicknessProperty;
+setter3.Value = 1;
+style1.Setters.Add(setter3);
+
+//UnratedItemStyle
+Style style2 = new Style(typeof(Path));
+Setter setter4 = new Setter();
+setter4.Property = Path.FillProperty;
+setter4.Value = new SolidColorBrush(Colors.LightGray);
+style2.Setters.Add(setter4);
+
+Setter setter5 = new Setter();
+setter5.Property = Path.StrokeProperty;
+setter5.Value = new SolidColorBrush(Colors.LightGray);
+style2.Setters.Add(setter5);
+
+Setter setter6 = new Setter();
+setter6.Property = Path.StrokeThicknessProperty;
+setter6.Value = 1;
+style2.Setters.Add(setter6);
+
+SfRating rating = new SfRating();
+
+rating.Value = 3;
+rating.ItemsCount = 5;
+rating.RatedItemStyle = style1;
+rating.UnratedItemStyle = style2;
+
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI Rating control with style](Rating_images/winui_rating_style.png)
+
+## ItemSize 
+
+The Rating control itemsize can be customized using the `ItemSize` Property. The default value is 24.
+
+{% tabs %}
+{% highlight XAML %}
+
+<syncfusion:SfRating Value="3"
+     ItemsCount="5" ItemSize="50">
+</syncfusion:SfRating>
+
+{% endhighlight %}
+{% highlight C# %}
+
+SfRating rating = new SfRating();
+
+rating.Value = 3;
+rating.ItemsCount = 5;
+rating.ItemSize = 50;
+
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI Rating control with itemsize](Rating_images/winui_rating_itemsize.png)
+
+## IsReadOnly
+
+The Rating control isreadonly can be customized using the `IsReadOnly` Property. The default value is `false`.
+
```

Comment:
@Priyanka-Vijayasankar The Rating control provides support to restrict the user's selection and only allows viewing of the rating value by setting the `IsReadOnly` property value as true. The default value is `false`.

Reply:
Changed the content as mentioned above.